README.md

```diff
@@ -1 +1,26 @@
-# Red-Policies
\ No newline at end of file
+# Red-Policies
+Contains all proposed policies for Cog-Creators/Red-DiscordBot
+
+The [definitions and clarifications](definitions.md) for terms used in these policies can be seen here. 
+
+## [Active policies](policies/accepted)
+- [Policy 1](policies/accepted/1.md) - Creation and modification of proposals
+- [Policy 2](policies/accepted/2.md) - Org Teams
+- [Policy 3](policies/accepted/3.md) - Core Devs
+- [Policy 4](policies/accepted/4.md) - QA
+- [Policy 5](policies/accepted/5.md) - Official Support Server (Server restructure)
+- [Policy 6](policies/accepted/6.md) - Disputes and Disagreements
+- [Policy 7](policies/accepted/7.md) - Voting timelines
+- [Policy 8](policies/accepted/8.md) - Permanent owner status for Twentysix
+
+### <ins>Policies Votes</ins>
+| Policies | Status | Votes in Favour | Votes Against | Votes Abstained | Votes Not Cast | Total Available votes | Date |
+|:---|---|:---:|:---:|:---:|:---:|:---:|:---:|
+| [Policy 1](policies/accepted/1.md) | *passed* | 11 | 0 | 3 | 2 | 16 | 01/08/2020
+| [Policy 2](policies/accepted/2.md) | *passed* | 11 | 0 | 3 | 2 | 16 | 01/08/2020
+| [Policy 3](policies/accepted/3.md) | *passed* | 11 | 0 | 3 | 2 | 16 | 01/08/2020
+| [Policy 4](policies/accepted/4.md) | *passed* | 11 | 0 | 3 | 2 | 16 | 01/08/2020
+| [Policy 5](policies/accepted/5.md) | *passed* | 9 | 2 | 3 | 2 | 16 | 01/08/2020
+| [Policy 6](policies/accepted/6.md) | *passed* | 11 | 0 | 3 | 2 | 16 | 01/08/2020
+| [Policy 7](policies/accepted/7.md) | *passed* | 11 | 0 | 3 | 2 | 16 | 01/08/2020
+| [Policy 8](policies/accepted/8.md) | *passed* | 7 | 0 | 1 | 0 | 8 | 02/08/2020
```

Comment:
Dates should follow ISO 8601 in order to be unambiguous between Americans / non Americans.